check-before-deployment.md

```diff
@@ -0,0 +1,342 @@
+
+    ```bash
+    sudo systemctl status firewalld.service
+    ```
+
+## Check and install the NTP service
+
+TiDB is a distributed database system that requires clock synchronization between nodes to guarantee linear consistency of transactions in the ACID model.
+
+At present, the common solution to clock synchronization is to use the Network Time Protocol (NTP) services. You can use the `pool.ntp.org` timing service on the Internet, or build your own NTP service in an offline environment.
+
+To check whether the NTP service is installed and whether it synchronizes with the NTP server normally, take the following steps:
+
+1. Run the following command. If it returns `running`, then the NTP service is running.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    sudo systemctl status ntpd.service
+    ```
+
+    ```
+    ntpd.service - Network Time Service
+    Loaded: loaded (/usr/lib/systemd/system/ntpd.service; disabled; vendor preset: disabled)
+    Active: active (running) since 一 2017-12-18 13:13:19 CST; 3s ago
+    ```
+
+2. Run the `ntpstat` command to check whether the NTP service synchronizes with the NTP server. 
+
+    > **Note:**
+    >
+    > For the Ubuntu system, you need to install the `ntpstat` package.
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    ntpstat
+    ```
+
+    - If it returns `synchronised to NTP server` (synchronizing with the NTP server), then the synchronization process is normal.
+
+        ```
+        synchronised to NTP server (85.199.214.101) at stratum 2
+        time correct to within 91 ms
+        polling server every 1024 s
+        ```
+
+    - The following situation indicates the NTP service is not synchronizing normally:
+
+        ```
+        unsynchronised
+        ```
+
+    - The following situation indicates the NTP service is not running normally:
+
+        ```
+        Unable to talk to NTP daemon. Is it running?
+        ```
+
+To make the NTP service start synchronizing as soon as possible, run the following command. Replace `pool.ntp.org` with your NTP server.
+
+{{< copyable "shell-regular" >}}
+
+```bash
+sudo systemctl stop ntpd.service && \
+sudo ntpdate pool.ntp.org && \
+sudo systemctl start ntpd.service
+```
+
+To install the NTP service manually on the CentOS 7 system, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+sudo yum install ntp ntpdate && \
+sudo systemctl start ntpd.service && \
+sudo systemctl enable ntpd.service
+```
+
+## Manually configure the SSH mutual trust and sudo without password
+
+This section describes how to manually configure the SSH mutual trust and sudo without password. It is recommended to use TiUP for deployment, which automatically configure SSH mutual trust and login without password. If you deploy TiDB clusters using TiUP, ignore this section.
+
+1. Log in to the Target Machine respectively using the `root` user account, create the `tidb` user and set the login password.
```

Comment:
```suggestion
1. Log in to the target machine respectively using the `root` user account, create the `tidb` user and set the login password.
```
s/Target Machine/target machine in batch @ran-huang

Reply:
@yikeke Personally I think either uppercase or lowercase is okay. Now we use mostly `Control Machine` and `target machine` in our docs. Better unify them.

Reply:
We take the lowercase form then. Can you help unify the terms across our docs in this PR?

Reply:
Sure.